include/mata/nfa.hh

```diff
@@ -284,6 +284,36 @@ public:
      */
     bool empty() const;
 
+    /**
+     * @brief Append post vector to the delta.
+     * 
+     * @param post_vector Vector of posts to be appended.
+     */
+    void append(const std::vector<Post>& post_vector) {
```

Comment:
In fact it cannot break sortedness. However, it may have an unexpected behaviour.

Reply:
The index of first post in the appended `post_vector` will be changed to index `size()` in delta. I agree that if this is the intended behaviour, it cannot break sortedness. But it is not the same as merging two deltas together. We should therefore clarify this different. I already assumed wrongly about these indices after the append is performed.

Reply:
It is a different thing imo. The method do not claim that it appends two Deltas.

Reply:
Yeah, it is. I just confused the two together because we talked earlier about having a function to merge two deltas. OK, you convinced me that it is not necessary to explain here any further.